rpc/shared/shared/src/main/scala/io/udash/rpc/AutoUdashRPCFramework.scala

```diff
@@ -0,0 +1,16 @@
+package io.udash.rpc
+
+import com.avsystem.commons.serialization._
+
+import scala.language.postfixOps
+
+trait AutoUdashRPCFramework extends UdashRPCFramework {
+  type RawValue = String
+
+  implicit val RawValueCodec: GenCodec[RawValue] =
```

Comment:
This is minor, but I think it's a problem I've run into at least a few times and I think there is slightly better approach to this:

If base trait needs an implicit, don't declare its abstract method as `implicit`. Instead, create a private method that delegates to abstract method and make the private one `implicit`.
Why? Because you could then do this:

```scala
val RawValueCodec = GenCodec[String]
```
and you don't need to point explicitly to `GenCodec.StringCodec`.